src/components/InLineNotification.tsx

```diff
@@ -81,7 +81,6 @@ const styles = StyleSheet.create({
   container: {
     padding: Spacing.Regular16,
     borderRadius: Spacing.Regular16,
-    width: '100%',
```

Comment:
This was causing the notification to overflow if margins are used in the style prop. This was actually not required, I've tested every usage of this including
- SendEnterAmount
- SwapScreen
- TokenImport
- ActionRequest
- SessionRequest

Reply:
Double checked on ActionRequest and LGTM.